bankdroid-interface/src/main/java/com/liato/bankdroid/api/configuration/Field.java

```diff
@@ -0,0 +1,82 @@
+package com.liato.bankdroid.api.configuration;
+
+import java.util.List;
+
+/**
+ * Represent an input field that is rendered for the user.
+ */
+public interface Field {
+
+    /**
+     * Returns the reference to this field. This value needs to be unique per configuration.
+     *
+     * @return the reference to this field.
+     */
+    String getReference();
+
+    /**
+     * Get the placeholder for this field.
+     * The placeholder specifies a short hint that describes the expected value of an
+     * field (e.g. a sample value or a short description of the expected format).
+     * The short hint is displayed in the input field before the user enters a value.
+     *
+     * @return the placeholder for this field.
+     */
+    String getPlaceholder();
+
+    /**
+     * Get the label for this field.
+     *
+     * @return the label for this field.
+     */
+    String getLabel();
+
+    /**
+     * Get the field type for this field.
+     *
+     * @return The field type for this field. Defaults to {@link FieldType#TEXT}.
+     */
+    FieldType getFieldType();
+
+    /**
+     * Returns {@code true} if the field is a required field.
+     *
+     * The {@link #validate(String)} will fail if this value is true and the string to be validated
+     * are empty or {@code null}.
+     * @return {@code true} if the field is a required field. Otherwise {@code false}.
+     */
+    boolean isRequired();
+
+    /**
+     * Returns {@code true} if the field should be hidden for the end user when it is rendered.
+     *
+     * @return {@code true} if the field should be hidden. Otherwise {@code false}.
+     */
+    boolean isHidden();
+
+    /**
+     * Returns {@code true} if, and only if, the field value should be encrypted before it is
+     * stored. An encrypted field should be treated as a sensitive field and hence be rendered as a
+     * password field.
+     * @return {@code true} if the field value should be encrypted before it is stored. Otherwise
+     * {@code false}.
+     */
+    boolean isEncrypted();
```

Comment:
The diagram in #534 names this property `secured` instead of `encrypted`. Do you @liato @fredrike have any comments on the naming to be used here? I do not want to use `password` because this could represent fields that are not passwords. Another suggestion is to use `secret`